code/src/Components/SecondQuestion.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Button from './Button';
+
+const SecondQuestion = ({ userGender, onGenderChange, section, setSection }) => {
+
+  return (
+    <section className="question-section">
+      <h2 className="question-heading">Vilket kön tillhör du?</h2>
+      <select 
+        value={userGender}
+        onChange={e => onGenderChange(e)}
+      >
+        <option value="">Välj</option>
+        <option value="kvinna">Kvinna</option>
+        <option value="man">Man</option>
+        <option value="annat">Annat</option>
+      </select>
+      <Button 
+        disable={onGenderChange} 
```

Comment:
Disablebutton doesn't work, try to set {userGender === ""}

Reply:
Couldn't get it to work, took another route :-)